### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by trisDeveloper**
Adding the home page

### CODE REVIEW

File: frontend/src/components/homepage/homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                  </div>
+                </div>
+              </div>
+              <div className="role p-3 w-full max-w-[400px] md:w-[45%] rounded-[40px] bg-[#fa974663]">
+                <div className="image md:w-full max-h-[200px] justify-center flex py-4">
+                  <img src={studentsImage} alt="help each other to learn" />
+                </div>
+                <div className="content w-full text-center">
+                  <h1 className="text-slate-900 font-black text-2xl tracking-tight text-center dark:text-white">
+                    Became A Tutor
```
Comment (marksist300) [inline frontend/src/components/homepage/homepage.jsx:92]: All looks good to me 🙂. Just a small typo to be-> 'Become A Tutor'

**Review by marksist300 — CHANGES_REQUESTED** [review]
Code looks good to me 👌🏻😊, nice work! Just a small typo on the links to choose the user type: line:109

**Comment by ermix3** [comment]
I think For its Good.

**Review by ermix3 — APPROVED** [review]
Good job.

**Review by theborgh — APPROVED** [review]
Looks good! :) 
As you see below, merging to develop is still blocked because there is a change requested (fixing a typo)

File: frontend/src/components/homepage/homepage.jsx (hunk 1 above)
Comment (theborgh) [reply frontend/src/components/homepage/homepage.jsx]: typo fixed, merging PR

**Pull request merged**